docs/src/experiment_setup_guide.md

```diff
@@ -0,0 +1,115 @@
+```julia
+using EnsembleKalmanProcesses.ParameterDistributions
+mean = 125
+standard_deviation = 40
+lower_bound = -50
+upper_bound = Inf
+constrained_gaussian("name", mean, standard_deviation, lower_bound, upper_bound)
+```
+This will output a ParameterDistribution in unconstrained space:
+```
+ParameterDistribution with 1 entries: 
+'name' with Constraint{BoundedBelow}[Bounds: (-50, ∞)] over distribution 
+Parameterized(Distributions.Normal{Float64}(μ=5.1393230339433345, σ=0.2256676316186263)) 
+```
+Copy the mean and standard deviation values into the constructor for the prior distribution in your TOML file like so:
+`prior = "Parameterized(Normal(5.1393230, 0.225667))"`
+
+You can also ensure that the bounds match the `constraint` entry in your TOML file. 
+Since `constraint` is a TOML string parsed to a vector in Julia, ensure that you have quotes around the square brackets for the constraint list.
+
+Constraint constructors:
+- Lower bound: `bounded_below(0)`
+- Upper bound: `bounded_above(2)`
+- Upper and lower bounds: `bounded(0, 2)`
+
+## Observation Map
+This component is inflexible for now. Your observation map file must be a script with the name `observation_map.jl` which exports a function `observation_map(iteration)`. 
+These requirements are because the observation map is computed in the update step like so:
+```julia
+include("experiments/$job_id/observation_map.jl")
+observation_map(iteration)
+```
+This function must load in model diagnostics for each ensemble member in the iteration and construct an array `arr = Array{Float64}(undef, dims..., ensemble_size)` such that 
+`arr[:, i]` will return the i-th ensemble member's observation map output.
+
+In the update step of EKI, the array will be saved in a JLD2 file named `observation_map.jld2` in the iteration folder of the output directory.
+
+If you are running a perfect-model experiment and generating truth data from ClimaAtmos itself, you may find it useful to create a kernel function to compute the observation map for each ensemble member. You can use this to run the default simulation's diagnostic output through the observation map and save the truth data and noise. An example of this is found in `experiments/sphere_held_suarez_rhoe_equilmoist/observation_map.jl`
+
+## Generating Truth Data
+The truth data must be a vector of observations with the same dimensions as an individual ensemble member entry in the observation map (`arr[:, i]` from above).
+
+The noise should be the covariance matrix of the full data that the truth comes from.
+
+Save each of these to a JLD2 file in the experiment folder. These will be read in when constructing the EKP object.
```

Comment:
Do you save all the samples? or just 1. Be very specific here, also include the file names.